examples/weather/modules/weather-data-l1/src/main/scala/my/metagraph/data/l1/WeatherDataL1App.scala

```diff
@@ -0,0 +1,17 @@
+package my.metagraph.data.l1
+
+import my.weathermetagraph.buildinfo.BuildInfo
+import org.tessellation.currency.l1.CurrencyL1App
+import org.tessellation.schema.cluster.ClusterId
+import org.tessellation.schema.semver.{MetagraphVersion, TessellationVersion}
+
+import java.util.UUID
+
+object WeatherDataL1App
```

Comment:
since the ML0 nodes have to do all the blockchain operations and are submitting the updates to the chain anyway, why not make the Data-L1 nodes responsible for fetching data from the external APIs? This way they are doing a little more than just presenting an endpoint for accepting updates.